applications/dashboard/models/class.usermodel.php

```diff
@@ -2932,18 +2932,24 @@ public function saveIP($userID, $IP, $dateUpdated = false) {
 
     /**
      * Updates visit level information such as date last active and the user's ip address.
-     *
-     * @param int $UserID
-     * @param string|int|float $ClientHour
+     * @param $UserID
+     * @param int|float $ClientHour
+     * @throws Exception
+     * @return bool True on success false on failure
```

Comment:
There's actually very specific conditions for false here, so this could be misleading.  How about...

```
* @return bool False if the user is banned or deleted, otherwise true.
```